easybuild/easyconfigs/q/Qhull/Qhull_pkgconfig.patch

```diff
@@ -0,0 +1,18 @@
+inspired by qhull-2003.1-pkgconfig.patch, available from git://pkgs.fedoraproject.org/qhull
```

Comment:
well, for this case, it's quite trivial:
```cmake
CONFIGURE_FILE(
  "${CMAKE_CURRENT_SOURCE_DIR}/qhull.pc.in"
  "${CMAKE_CURRENT_BINARY_DIR}/qhull.pc"
)
INSTALL(FILES "${CMAKE_BINARY_DIR}/qhull.pc"  DESTINATION lib/pkgconfig)
```
Use `PROJECT_VERSION` and `CMAKE_INSTALL_PREFIX` in the `.pc.in` file.